src/LogExpert/Config/ConfigManager.cs

```diff
@@ -137,7 +137,7 @@ private Settings Load ()
             Directory.CreateDirectory(dir);
         }
 
-        if (!File.Exists(dir + Path.DirectorySeparatorChar + "settings.json"))
+        if (!File.Exists(Path.Combine(dir, "settings.json")))

```

Comment:
Path.Combine, sometimes does not work with linux and can have unforeseen consequences => https://learn.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=net-9.0#remarks

Reply:
I don't see the Linux part you are talking about in the link. Can you point me to it?
Reading a bit about the new Path.Join I still see no reason to not use Path.Combine. It is better than the current code and gives a better result than Path.Join in case there is an absolute path.